.github/ISSUE_TEMPLATE/4-report-a-flaky-test.yml

```diff
@@ -35,7 +35,8 @@ body:
         - macOS ARM64
         - macOS x64
         - SmartOS
-        - Windows
+        - Windows x64
+        - Windows ARM64
```

Comment:
Nit: this list, I believe, was supposed to be sorted alphabetically, which would put Arm64 before x64. Just like the macOS.